scene/3d/xr_hand_modifier_3d.h

```diff
@@ -65,6 +60,17 @@ class XRHandModifier3D : public Node3D {
 protected:
 	static void _bind_methods();
 
+	virtual void _skeleton_changed(Skeleton3D *p_old, Skeleton3D *p_new) override;
+	virtual void _process_modification(double p_delta) override;
+
+#ifndef DISABLE_DEPRECATED
+	bool _set(const StringName &p_name, const Variant &p_value);
+	bool _get(const StringName &p_name, Variant &r_ret) const;
+	void _set_target_bind_compat_87888(const NodePath &p_target);
+	NodePath _get_target_bind_compat_87888() const;
+	static void _bind_compatibility_methods();
+#endif // DISABLE_DEPRECATED
```

Comment:
If this PR gets merged before Godot 4.3, then we don't need any of this compatibility stuff because `XRHandModifier3D` is new for Godot 4.3. So, this stuff is only needed if this PR is going to be merged for Godot 4.4 or later.